go/libraries/doltcore/sqle/dtablefunctions/dolt_log.go

```diff
@@ -515,41 +530,34 @@ func (ltf *LogTableFunction) RowIter(ctx *sql.Context, row sql.Row) (sql.RowIter
 	return ltf.NewDotDotLogTableFunctionRowIter(ctx, sqledb.DbData().Ddb, commits, notCommits, matchFunc, cHashToRefs, ltf.tableNames)
 }
 
-// evaluateArguments returns revisionValStrs, notRevisionValStrs, and three dot boolean.
-// It evaluates the argument expressions to turn them into values this LogTableFunction
-// can use. Note that this method only evals the expressions, and doesn't validate the values.
-func (ltf *LogTableFunction) evaluateArguments() (revisionValStrs []string, notRevisionValStrs []string, threeDot bool, err error) {
-	for _, expr := range ltf.revisionExprs {
-		valStr, err := expressionToString(ltf.ctx, expr)
-		if err != nil {
-			return nil, nil, false, err
-		}
-
-		if strings.Contains(valStr, "..") {
-			if strings.Contains(valStr, "...") {
-				refs := strings.Split(valStr, "...")
-				return refs, nil, true, nil
-			}
-			refs := strings.Split(valStr, "..")
-			return []string{refs[1]}, []string{refs[0]}, false, nil
+// parseRevisionRange processes a revision string for range syntax (.., ...)
+// Returns (revisions, notRevisions, threeDot, isRange)
+func parseRevisionRange(revisionStr string) ([]string, []string, bool, bool) {
+	if strings.Contains(revisionStr, "..") {
+		if strings.Contains(revisionStr, "...") {
+			refs := strings.Split(revisionStr, "...")
+			return refs, nil, true, true
 		}
-
-		revisionValStrs = append(revisionValStrs, valStr)
+		refs := strings.Split(revisionStr, "..")
+		return []string{refs[1]}, []string{refs[0]}, false, true
 	}
+	return nil, nil, false, false
+}
 
-	for _, notExpr := range ltf.notRevisionExprs {
-		notValStr, err := expressionToString(ltf.ctx, notExpr)
-		if err != nil {
-			return nil, nil, false, err
-		}
-
-		if strings.HasPrefix(notValStr, "^") {
-			notValStr = strings.TrimPrefix(notValStr, "^")
+// evaluateArguments returns revisionValStrs, notRevisionValStrs, and three dot boolean.
+// It processes the stored revision strings from the CLI parser and handles range syntax.
+func (ltf *LogTableFunction) evaluateArguments() (revisionValStrs []string, notRevisionValStrs []string, threeDot bool, err error) {
```

Comment:
It's a little messy to have a function called `evaluateArguments()` and another function in the same type called `evalArguments()`. That makes it hard for readers to understand the difference and when to use each. See if you can clean that up and give each a good, self-documenting name that makes them easy to distinguish, even if someone hasn't read the function documentation.